tests/Client/Cursor.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Cursor', () => {
     });
 
     it('accepts non-default things', () => {
-      expect(new Cursor('channel', {limit: 50})).to.eql({
+      expect(new Cursor('channel', { limit: 50 as any })).to.eql({
```

Comment:
Fix here is not to do `50 as any`, but that `limit` should be defined as a `number` (instead of `null | undefined`).